opcua/common/subscription.py

```diff
@@ -334,3 +334,50 @@ def get_event_properties_from_type_node(node):
         curr_node = parents[0]
 
     return properties
+
+
+def modify_monitored_item(self, handle, new_samp_time, new_queuesize = 0, mod_filter = None):
+    """
+    Modify a monitored item.
+    :param handle: Handle returned when originally subscribing
+    :param new_samp_time: New wanted sample time
+    :param new_queuesize: New wanted queuesize, default is 0
+    :param mod_filter: New wanted filter, default is None
+    :return: Return a Modify Monitored Item Result
+    """
+    modif_item = ua.MonitoredItemModifyRequest()
+    modif_item.MonitoredItemId = handle
+    modif_item.RequestedParameters = self._modify_monitored_item_request(new_queuesize, new_samp_time, mod_filter)
+    params = ua.ModifyMonitoredItemsParameters()
+    params.SubscriptionId = self.subscription_id
+    params.ItemsToModify.append(modif_item)
+    results = self.server.modify_monitored_items(params)
+    return results
+
+
+def _modify_monitored_item_request(self, new_queuesize, new_samp_time, mod_filter):
+    req_params = ua.MonitoringParameters()
+    with self._lock:
+        req_params.ClientHandle = self._client_handle
+    req_params.QueueSize = new_queuesize
+    req_params.Filter = mod_filter
+    req_params.SamplingInterval = new_samp_time
+    return req_params
+
+
+def deadband_monitor(self, var, deadband_val, deadbandtype = 1, queuesize = 0, attr=ua.AttributeIds.Value):
+    """
+    Function to create a subscription with a Deadband Value.
+    Default deadband value type is absolute.
```

Comment:
this is function(should be a method) to create a monitored item or to subscribe to data changes with deadband